### ISSUE TITLE
NTR: Brain biopsy: globoid cells in white matter (Lab: pathology: biopsies & morphology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9130]
New term request Brain biopsy: globoid cells in white matter (segal_100513194153):   UMLS: 2237952

**Comment by pnrobinson** [comment]
Multinucleated phagocytes in the cerebral white matter, typically are 10-15 micrometers in diameter, that contain fatty filamentous inclusions that stain positive for periodic acid-Schiff (PAS), which may be phagocytized myelin, as these cells are often found in proximity to sites where demyelination is prominent. Globoid cells have been to thought to originate from CNS resident microglia and/or infiltrating peripheral macrophages.  PMID:27638591

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000892
- label: White matter globoid cells
- parents: [Abnormal cerebral white matter morphology (HP:0002500)](https://hpo.jax.org/app/browse/term/HP:0002500)
- pmid: PMID:27638591
- def: Multinucleated phagocytes in the cerebral white matter, typically are 10-15 micrometers in diameter, that contain fatty filamentous inclusions that stain positive for periodic acid-Schiff (PAS), which may be phagocytized myelin, as these cells are often found in proximity to sites where demyelination is prominent. Globoid cells have been to thought to originate from CNS resident microglia and/or infiltrating peripheral macrophages.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9130
- synonyms:
